README.md

```diff
@@ -458,6 +459,13 @@ There are 3 main table migration workflows, targeting different table types. All
   - Migrate tables using CTAS 
   - Experimentally migrate Delta and Parquet data found in dbfs mount but not registered as Hive Metastore table into UC tables.
 
+### Post Migration Data Reconciliation Task
+UCX also provides `migrate-data-reconciliation` workflow to validate the integrity of the migrated tables:
+- Compare the schema of the source and target tables. 
+- Compare the row counts of the source and target tables.
+- Compare the content of individual row between source and target tables to identify any discrepancies.
+Once completed, the output will be stored in `$inventory_database.recon_result` Delta table.
```

Comment:
can you attach the screenshot with the sample result and short explanation on how to interpret it?..